### ISSUE TITLE
minitube-2.7 search on channel/keyword gives blackscreen

### ISSUE THREAD

**Issue by juanitotc** [flaviotordini/minitube#81]
I had previously compiled minitube-2.5 and had it working without problems.

I've just compiled minitube-2.7, but when searching on channel/keyword I get a black screen and:

https://www.googleapis.com/youtube/v3/search?&part=snippet&type=video&maxResults=50&q=test&safeSearch=strict 0

If I enter this into a browser (minus the trailing zero), I get the text pasted below.

I hardly ever use minitube - what causes this error?

{
 "error": {
  "errors": [
   {
    "domain": "usageLimits",
    "reason": "dailyLimitExceededUnreg",
    "message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup.",
    "extendedHelp": "https://code.google.com/apis/console"
   }
  ],
  "code": 403,
  "message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup."
 }
}

**Comment by flaviotordini** [comment]
Did set up an API key? Check the README

**Comment by juanitotc** [comment]
Yes I did set up an API key - I compiled with "make V=1" so I could see it repeated endlessly.

Just in case, I also started minitube with GOOGLE_API_KEY="blah-blah" minitube

**Comment by flaviotordini** [comment]
Did you enable the YouTube Data API in Google dev console?

**Comment by juanitotc** [comment]
When I add the key parameter in a browser, things seem to work;

kind	"youtube#searchListResponse"
etag	"\"m2yskBQFythfE4irbTIeOgYYfBU/Dus9GWOM8oNUvwaYsrv8tzpDPWo\""
nextPageToken	"CDIQAA"
regionCode	"FR"
pageInfo	
totalResults	1000000
resultsPerPage	50
items	[50]

**Comment by flaviotordini** [comment]
When running from command line do you get any error message?

**Comment by juanitotc** [comment]
No, nothing at all :(

Are there some debug codes I can pass?

**Comment by Zauberfisch** [comment]
Did anyone figure this out?

My google dashboard now shows that my quata limit is 0/requests per day, and when I want to change it I get:

> Enter a number between 0 and 0. To increase this maximum, apply for higher quota (https://support.google.com/youtube/contact/yt_api_form?hl=en_GB).

Is there some know way to get this quata extended without having to apply for it?
